packages/eslint-plugin-calcite-components/src/rules/require-event-emitter-type.ts

```diff
@@ -1,5 +1,4 @@
 import { Rule } from "eslint";
-// @ts-ignore
```

Comment:
PSA: do not use `// @ts-ignore` as it's dangerous.
Reason: once the thing that was causing the TypeScript error is fixed, the `// @ts-ignore` comment may remain in the code. In the future, if another TypeScript error may occur in that place, you won't see it because `// @ts-ignore` would hide it.

Safer alternative: `// @ts-expect-error` - acts just like `// @ts-ignore`, except, if no error was reported, TypeScript will complain of needless `// @ts-expect-error` (just like it was needless in this case it seems like)